Documentation/guides/quick_start.md

```diff
@@ -2,77 +2,64 @@
 
 ## Installing HARK
 
-HARK is an open source project that is compatible with Python 3.
-
-### Installing HARK with pip
+HARK is an open source project that is compatible with Python 3. Currently, we recommend using version 3.10 or lower.
 
 The simplest way to install HARK is to use [pip](https://pip.pypa.io/en/stable/installation/).
 
-To install HARK with pip, at a command line type `pip install econ-ark`.
-
-(If you want to install a release that is not the default stable release, for instance if you want to install a development release, you'll need to use a "pinned" release number: `pip install econ-ark==0.10.1.dev1`, substituting "0.10.1.dev1" for your desired release number.)
-
-If you are installing via pip, we recommend using a virtual environment such as [virtualenv](https://virtualenv.pypa.io/en/latest/). Creation of a virtual environment isolates the installation of `econ-ark` from the installations of any other python tools and packages.
+Before installing HARK, we recommend creating a new virtual environment, which isolates the installation of `econ-ark` from the installations of any other Python tools and packages, thus avoiding conflicts.
 
-To install `virtualenv`, then to create an environment named `econ-ark`, and finally to activate that environment:
+To create a new virtual environment, enter this in your command line:
```

Comment:
I think we ought recommend `conda`, perhaps replace the rest of this sub-section with the following?

````markdown
To create a new environment, follow the steps below in your command line:

1. We first set up a Conda 'environment' for HARK.
   This is a way of isolating the installation of HARK so that it does not interfere
   with any other Python scripts or modules on your computer.
   To do so, at a command line type:

   ```console
   $ conda create -n econ-ark "python=3.10"
   ```

2. Activate the environment:

   ```console
   $ conda activate econ-ark
   ```

3. Install HARK into the environment:

   ```console
   (econ-ark) $ conda install econ-ark
   ```
````

I'd also consider remove the line about installing a non-default release, I'm unsure if it fits in the quick-start guide.